hack/test-env.sh

```diff
@@ -19,3 +19,6 @@ export ISTIO_VERSION="1.22.0"
 export ISTIO_UNSUPPORTED_E2E_TESTS="retry,httpoption"
 export CONTOUR_VERSION="v1.29.0"
 export CONTOUR_UNSUPPORTED_E2E_TESTS="httpoption"
+
+export ENVOY_GATEWAY_VERSION="latest"
+export ENVOY_GATEWAY_UNSUPPORTED_E2E_TESTS="httpoption"
```

Comment:
lets disable host rewrite for now ? with a comment saying EG doesnt support `ExternalName` Service routing ?